plugins/inputs/kube_inventory/node.go

```diff
@@ -14,15 +14,28 @@ func collectNodes(ctx context.Context, acc telegraf.Accumulator, ki *KubernetesI
 		acc.AddError(err)
 		return
 	}
+
+	ki.gatherNodeCount(len(list.Items), acc)
+
 	for _, n := range list.Items {
 		ki.gatherNode(n, acc)
 	}
 }
 
+func (ki *KubernetesInventory) gatherNodeCount(count int, acc telegraf.Accumulator) {
+	fields := map[string]interface{}{}
+	tags := map[string]string{}
+	fields["count"] = count
```

Comment:
What do you think about calling this 'node_count' to be more specific?

Reply:
Sounds good. I will modify it to node_count.